### DISCUSSION TITLE
Be able to tell easily when client is connected/disconnected (controlled side)

### DISCUSSION THREAD

**Discussion by bilogic** [rustdesk/rustdesk#3750] [Ideas]
### Description

Show a toast/message when client is disconnected

### Impact

Sometimes, I'm not sure if remote side has disconnected from my desktop, so have to check first before I can open more sensitive files.

**Comment by rustdesk** [comment]
there is timeout, after timeout it will pop message.

**Comment by bilogic** [comment]
Hmm, this is not about timeout.

1. Let's say A connect to B to see B's desktop.
2. How does B know if A is still connected or not?
3. Teamviewer likes to show promotion after A disconnect
4. My request is for RustDesk to show a message after A disconnect (for any reason), so that B knows that A is gone, e.g. 

> ## XXX has disconnected from RustDesk
> - [HH:MM:SS] Session started 
> - [HH:MM:SS] Session lasted for H hours M minutes

Thank you.

**Comment by rustdesk** [comment]
Does AnyDesk show?

**Comment by bilogic** [comment]
Sorry, I don't use AnyDesk, but I think it is helpful. Teamviewer also hides wallpaper by default, which is also a clear sign that somebody is connected.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/918
